viahtml/hooks/hooks.py

```diff
@@ -18,6 +19,14 @@
 ]
 
 
+def query_param_as_bool(value: str) -> bool:
```

Comment:
True. It might make sense to extract `via.app.asbool` into a utility module and share that in future, although has some behaviors that might be surprising in a query string - it treats "2" as falsey for example, whereas I'd expect any non-zero value to be truthy.